Source/ACE.Server/Config.json.example

```diff
@@ -16,7 +16,8 @@
     "DatFilesDirectory": "c:\\ACE\\",
     "ShutdownInterval": "60",
     "WorldDatabasePrecaching": false,
-    "LandblockPreloading": false
+    "LandblockPreloading": false,
+    "GlobalEventLandblockPreloading": true
```

Comment:
I realize that this is a rather simple change and you just want to get this out as quick as possible, but this is really duplicating the work of the existing LandblockPreloading system. Instead of fixing / improving the existing system so that it is using a config file, it is ignoring and bypassing that system, and adding a duplicate version of that system to load 1 specific landblock.

This is adding more cruft to the code and making things more complicated than it should be, and will just have to be fixed later. I would rather just apply the proper fix for this now, and improve the existing LandblockPreloading system to use config files, as originally described in this comment:

// TODO: Change the RawLandblockId list used for preloading defined landblocks to some other, more easily-modified format, rather than a compiled uint array